src/MSBuild/XMake.cs

```diff
@@ -3476,7 +3476,7 @@ private static void DisplayCopyrightMessage()
             const string frameworkName = ".NET Framework";
 #endif
 
-            Console.WriteLine(ResourceUtilities.FormatResourceString("CopyrightMessage", ProjectCollection.Version.ToString(), frameworkName));
+            Console.WriteLine(ResourceUtilities.FormatResourceString("CopyrightMessage", ThisAssembly.AssemblyInformationalVersion, frameworkName));
```

Comment:
🎉

Reply:
`ThisAssembly` is not available during sourdebuild builds (as git versioning is turned off). There's a `THISASSEMBLY` symbol you can ifdef on :(

Reply:
Added